Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/test-settings/test-launch-dialog.component.ts

```diff
@@ -1,8 +1,14 @@
 ﻿import { Component } from "@angular/core";
+import { Test } from "../tests.model";
 
 @Component({
     moduleId: module.id,
     selector: 'test-launch-dialog',
     templateUrl: "test-launch-dialog.html"
 })
-export class TestLaunchDialogComponent { }
+export class TestLaunchDialogComponent {
+    copiedContent: boolean = true;
+    settingObject: Test = new Test();
```

Comment:
This is an object of Test Class to obtain the Link of the Test from the database. It is required to bind the Test Link in the respective field in the test-launch-dialog.html file.